app/views/components/docs/image_meta.yml

```diff
@@ -13,17 +13,30 @@ examples:
   default:
     data:
       src: https://assets.publishing.service.gov.uk/government/assets/history/buildings/larry-the-cat-a47549e08bdbc6cd0e3e042eea943f65b7a4590d95642586e51acb44bb2dcea2.jpg
-      meta:
-        alt_text: Larry the cat
-        caption: Larry, Chief Mouser to the Cabinet Office
-        credit: Her Majesty's Government
+      alt_text: Larry the cat
```

Comment:
Do you know why the alt text is passed to the component twice?  I know you haven't introduced it in this change, but I'm curious as to why.

Reply:
It's because the component renders an image, with alt_text; and some metadata lines, which happen to include the alt text. The previous version tried to magically copy them across, but that's not compatible with the new, more flexible version.